solr/core/src/test/org/apache/solr/cloud/CollectionPropsTest.java

```diff
@@ -45,6 +45,7 @@
 @LuceneTestCase.Slow
 @SolrTestCaseJ4.SuppressSSL
 public class CollectionPropsTest extends SolrCloudTestCase {
+  public static final int TIMEOUT = 5000;
```

Comment:
might `private` be sufficient?
```suggestion
  private static final int TIMEOUT = 5000;
```